src/components/Modal.tsx

```diff
@@ -21,26 +21,26 @@ export const Modal: React.FC<ModalProps> = ({
   children,
   "data-cy": dataCy,
 }) => (
-  <StyledModal
-    maskStyle={{
-      backgroundColor,
-    }}
-    centered
-    footer={footer}
-    visible={visible}
-    onCancel={onCancel}
-    width="50%"
-    wrapProps={{
-      "data-cy": dataCy,
-    }}
-  >
-    {/* Placing a stop propogation event handler around this modal to prevent click events from bubbling up to their parent component  */}
-    {/* eslint-disable-next-line */}
-    <div onClick={(e) => e.stopPropagation()}>
+  // Placing a stop propogation event handler around this modal to prevent click events from bubbling up to their parent component
```

Comment:
this diff moves the stopPropapation call to wrap the entire modal so we can capture clicks on the modal background